app/models/transfer/task_list.rb

```diff
@@ -5,7 +5,8 @@ def self.layout
         identifier: :project_kick_off,
         tasks: [
           Transfer::Task::HandoverTaskForm,
-          Transfer::Task::StakeholderKickOffTaskForm
+          Transfer::Task::StakeholderKickOffTaskForm,
+          Transfer::Task::MasterFundingAgreementTaskForm
```

Comment:
This goes into a section called `identifier: :legal_documents,` as per Conversions (it's not obvious from the ticket tho!)